src/Dashboard/Properties/AssemblyInfo.cs

```diff
@@ -1,10 +1,6 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System.Reflection;
 using System.Runtime.CompilerServices;
 
-[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
Nice to remove all this duplication!

Reply:
Yeah, I was thinking about that as well -- I'll see if there's a straightforward way to get this working there when I tackle that repo next.

Reply:
Actually, I think these should all be 2.2.0.0 - looks like I missed doing this when I revved the minor version last

Reply:
Doh! You're right. Good catch, I didn't even notice that.